core/mobile-manager/src/backend/MobileAuthorizationBackend.ts

```diff
@@ -82,11 +76,11 @@ export class MobileAuthorizationBackend extends NativeAppAuthorizationBackend {
   }
 
   /** return accessToken */
-  public async refreshToken(): Promise<AccessToken> {
-    return new Promise<AccessToken>((resolve, reject) => {
+  public async refreshToken(): Promise<AccessTokenString> {
+    return new Promise<AccessTokenString>((resolve, reject) => {
       MobileHost.device.authGetAccessToken((tokenString?: string, err?: string) => {
         if (!err && tokenString) {
-          resolve(AccessToken.fromJson(JSON.parse(tokenString) as AccessTokenProps));
+          resolve(tokenString);
```

Comment:
Doesn't the JSON response from authGetAccessToken still need to be parsed, since AccessTokenString is expected to be the tokenString alone?

```suggestion
          resolve((JSON.parse(tokenString) as AccessTokenProps).tokenString);
```

If JSON is still being returned, could we also rename the `tokenString` argument in the callback to `tokenStringJson` or something similar to convey that it is not an AccessTokenString?

Reply:
Good catch, I've added that now.